games-strategy/freesynd/freesynd-0.8.recipe

```diff
@@ -10,15 +10,15 @@ as described here: https://community.pcgamingwiki.com/files/file/1201-syndicate-
 HOMEPAGE="https://freesynd.sourceforge.io/"
 COPYRIGHT="2024 The FreeSynd Project"
 LICENSE="GNU GPL v2"
-REVISION="1"
+REVISION="2"
```

Comment:
This is technically not needed because the 64 bit build was successful. When leaving this at 1, the 64 bit version will not be rebuilt.